client/components/App.js

```diff
@@ -14,7 +18,12 @@ const App = props => {
 
   return (
     <BrowserRouter>
-      <Route path="/pets" component={NavBar} />
+      <Switch>
+        <Route exact path="/pets" component={PetTypesList} />
+        <Route exact path="/pets/:type/:id" component={PetShow} />
+        <Route exact path="/pets/:type" component={AdoptablePetsOfAType} />
+        <Route exact path="/adoption/new" component={SurrenderForm} />
+      </Switch>
     </BrowserRouter>
```

Comment:
Are you getting rid of the navbar?

Reply:
No. Just trying to get the application to run again. These components got removed and are needed to run.